Vagrantfile

```diff
@@ -1,33 +1,14 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-Vagrant.configure(2) do |config|
-  config.vm.box = "ubuntu/bionic64"
-
-  config.vm.provision "shell", inline: <<-SHELL
-    set -e
-    set -x
-
-    apt-get -y update
-    apt-get install -y unzip git imagemagick mercurial python-pip curl
-
-    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python get-pip.py && rm -f get-pip.py
 
-    # Install sphinx
-    pip install -U sphinx==1.8.3
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
 
-    # Install sphinx theme from ArduPilot repository
-    pip install git+https://github.com/ArduPilot/sphinx_rtd_theme.git -UI
-
-    # and a youtube plugin:
-    pip install git+https://github.com/sphinx-contrib/youtube.git -UI
-
-    # and a vimeo plugin:
-    hg clone https://bitbucket.org/jdouglass/sphinxcontrib.vimeo
-    pushd sphinxcontrib.vimeo
-    python setup.py build
-    python setup.py install
-    popd
-
-  SHELL
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "ubuntu/bionic64"
+  config.vm.provider "virtualbox" do |vb|
+      vb.name = "ArduPilot_wiki"
```

Comment:
If I have more than one Wiki Repo in different folders on my machine and I evoke Vagrant from each folder, it creates more than one VM. Vagrant creates a VM for each repo with a different sufix.

Would this behavior cease with the VM name being explicit set?

I do not see it as an issue because it is not usual to be useful. I just want to understand if there are some new behaviour.

Reply:
it will use the same machine but change the directory in /vagrant to target the right repo where you launch the vagrant up. 
So yes, that the end of the multiple VM. The VM can still be updated with vagrant provision

Reply:
No problem I did use it only when changing stuffs on the Sphinx´s theme.  tks!